dm/master/server.go

```diff
@@ -1032,6 +1242,32 @@ func (s *Server) RefreshWorkerTasks(ctx context.Context, req *pb.RefreshWorkerTa
 	}, nil
 }
 
+// ShowMasterLeader implements MasterServer.ShowMasterLeader
+// Note: this request don't need save to etcd
+func (s *Server) ShowMasterLeader(ctx context.Context, req *pb.ShowMasterLeaderRequest) (*pb.ShowMasterLeaderResponse, error) {
+	log.L().Info("", zap.Stringer("payload", req), zap.String("request", "ShowMasterLeader"))
+
+	if s.election == nil {
+		return &pb.ShowMasterLeaderResponse{
+			Result: false,
+			Msg:    "no leader",
+		}, nil
+	}
+
+	_, leaderID, err := s.election.LeaderInfo(ctx)
```

Comment:
what‘s `leaderID` in DM?

Reply:
maybe it's better to add a function ` getDMLeaderMasterInfo`

Reply:
update in 20524e7